RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -577,6 +577,12 @@ boolean canYellowRay(monster target)
 		{
 			return true;
 		}
+		
+		// roman candelabra, also a 75 turn cooldown
+		if(auto_haveRoman())
```

Comment:
```suggestion
		if(auto_haveRoman() && auto_is_valid($skill[Blow the Yellow Candle\!]))
```
do we need to check whether you can equip the candelabra? e.g. you can't in Boris or Surprising Fist.